### PR TITLE
Add git plus package

### PR DESCRIPTION
**Pull request by radamant**
I've been using this for about an hour now and it already feels superior to atomatigit in functionality and stability. I committed and pushed this branch using git plus!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,10 @@ Key                 | Command
 &#x2303;S           | Fuzzy Search Forward
 &#x2303;R           | Fuzzy Search Reverse
 &nbsp;              |
-                    | **Git**
+                    | **Git (git plus)**
+&#x2318;&#x2318;H   | Open up git plus fuzzy find (try typing 'checkout' to see options)
```
Comment (aaronjensen) [inline README.md]: cmd-cmd-h?

Comment (radamant) [inline README.md]: Dang, I tried to push that fix before anyone could find that!

Comment (radamant) [inline README.md]: See previous comments about being annoyed at html entities :tongue:

**Comment by radamant** [comment]
git-plus

**Comment by aaronjensen** [comment]
nice!

**Pull request merged**